### ISSUE TITLE
AttributeError: 'SortImports' object has no attribute 'output'

### ISSUE THREAD

**Issue by jwhonce** [lexicalunit/atom-isort#17]
The following notification is presented every time I open a pane with a python file.

Unexpected errors while running python process.
Traceback (most recent call last):
  File ".../.atom/packages/atom-isort/src/isort-wrapper.py", line 114, in <module>
    IsortTools().read_and_process()
  File ".../.atom/packages/atom-isort/src/isort-wrapper.py", line 110, in read_and_process
    self._process_request(data)
  File ".../.atom/packages/atom-isort/src/isort-wrapper.py", line 64, in _process_request
    write_to_stdout=True).output
AttributeError: 'SortImports' object has no attribute 'output'

Running isort from the command line works without error.

Versions:
  Atom 1.20.1 x64 on RHEL 7.4
  atom-isort 2.5.2
  python 2.7.5
  virtualenvwrapper 4.7.2
  virtualenv 1.10.1

**Comment by lexicalunit** [comment]
@jwhonce do you have the text of the file you're trying to isort? This seems like it might be a bug in `isort` itself. There's differences in the way the `isort` command line tool works vs. the isort library, and that may be where their bug is.

**Comment by szferi** [comment]
I got the same error message as described above. After debugging the issue, I got the following warning:

> \_\_init\_\_.py was skipped as it's listed in 'skip' setting or matches a glob in 'skip_glob' setting

The ISortImports objects have no output attribute if the filename matches the skip settings.
And indeed the default settings of isort defines \_\_init\_\_.py as skip file:
 https://github.com/timothycrosley/isort/blob/88a23e8aaacddf727ce01040dab31b9f7f55c497/isort/settings.py#L50
As a workaround I changed line 62-64 in isort-wrapper.py from this:
```
new_contents = isort.SortImports(file_contents=request['file_contents'],
                                             file_path=request.get('file_path'),
                                             write_to_stdout=True).output

```
to this:
```
new_contents = isort.SortImports(file_contents=request['file_contents'],
                                             file_path=request.get('file_path'),
                                             write_to_stdout=True, 
                                             not_skip=['__init__.py']).output
```

**Comment by lexicalunit** [comment]
Thank you @szferi, updated the code and released as v2.6.0.

**Comment by lexicalunit** [comment]
Also patch v2.6.1 should prevent the error @jwhonce saw from ever happening again.
